examples/using-javascript-transforms/gatsby-node.js

```diff
@@ -83,6 +90,7 @@ exports.createPages = ({ graphql, boundActionCreators }) => {
             createPage({
               path: frontmatter.path, // required
               component: markdownTemplate,
+              layout: `index`,
```

Comment:
Not required to specify the index specifically as the index layout is set by default.